File: backend/xingyun-basedata/src/main/java/com/lframework/xingyun/basedata/vo/supplier/QuerySupplierVo.java
package com.lframework.xingyun.basedata.vo.supplier;

import com.lframework.starter.web.core.vo.BaseVo;
import com.lframework.starter.web.core.vo.SortPageVo;
import io.swagger.annotations.ApiModelProperty;
import java.io.Serializable;
import lombok.Data;

@Data
public class QuerySupplierVo extends SortPageVo implements BaseVo, Serializable {

  private static final long serialVersionUID = 1L;

  /**
   * 编号
   */
  @ApiModelProperty("编号")
  private String code;

  /**
   * 名称
   */
  @ApiModelProperty("名称")
  private String name;

  /**
   * 供应商启用状态；为空时查询全部状态。
   */
  @ApiModelProperty("供应商启用状态")
  private Boolean available;

}

File: frontend/src/api/base-data/supplier/model/querySupplierVo.ts
import { SortPageVo } from '@/api/model/sortPageVo';

export interface QuerySupplierVo extends SortPageVo {
  /**
   *
   */
  serialVersionUID: long;

  /**
   * 编号
   */
  code: string;

  /**
   * 名称
   */
  name: string;

  /**
   * 状态；true 表示启用，false 表示停用
   */
  available: boolean | '';
}
